Package.resolved

```diff
@@ -46,6 +46,15 @@
           "version": "0.4.0"
         }
       },
+      {
+        "package": "Plot",
+        "repositoryURL": "https://github.com/johnsundell/plot.git",
+        "state": {
+          "branch": "a12222bc4d25",
```

Comment:
weird branch name but ok 😆

Reply:
oh, it's just the revision short hash. Can we pin to something more stable?

Reply:
not really because thead is not on the latest version